integrations/tensorflow/compiler/BUILD

```diff
@@ -56,9 +56,26 @@ cc_library(
 
 cc_binary(
     name = "iree-tf-opt",
+    srcs = ["tf_opt_main.cc"],
     deps = [
         ":tensorflow",
+        "//integrations/tensorflow/compiler/dialect/tf_strings/ir:dialect",
+        "//integrations/tensorflow/compiler/dialect/tf_tensorlist/ir:tf_tensorlist_dialect",
+        "//iree/compiler/Conversion:init_conversions",
+        "//iree/compiler/Conversion/HLOToLinalg",
+        "//iree/compiler/Dialect/HAL/Conversion:Passes",
+        "//iree/tools:init_compiler_modules",
+        "//iree/tools:init_iree_passes_and_dialects",
+        "//iree/tools:init_mlir_passes_and_dialects",
+        "//iree/tools:init_targets",
+        "//iree/tools:init_xla_dialects",
         "//iree/tools:iree_opt_main",
```

Comment:
```suggestion
```

Seems this can be dropped since a source file was added.

Reply:
Thanks for catching. How does that even compile with two main methods....?

Reply:
Static archive semantics: it'll pick the first object containing the symbol it needs. Objects that don't provide a required symbol aren't part of the link.